tests/foreman/ui/test_contenthost.py

```diff
@@ -0,0 +1,207 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Content Hosts UI
+
+@Requirement: Content Host
+
+@CaseAutomation: Automated
+
+@CaseLevel: Acceptance
+
+@CaseComponent: UI
+
+@TestType: Functional
+
+@CaseImportance: High
+
+@Upstream: No
+"""
+from nailgun import entities
+from robottelo.cli.factory import (
+    setup_org_for_a_custom_repo,
+    setup_org_for_a_rh_repo,
+)
+from robottelo.constants import (
+    FAKE_0_CUSTOM_PACKAGE,
+    FAKE_0_CUSTOM_PACKAGE_NAME,
+    FAKE_1_CUSTOM_PACKAGE,
+    FAKE_1_CUSTOM_PACKAGE_NAME,
+    FAKE_2_CUSTOM_PACKAGE,
+    FAKE_0_CUSTOM_PACKAGE_GROUP,
+    FAKE_0_CUSTOM_PACKAGE_GROUP_NAME,
+    FAKE_0_YUM_REPO,
+    PRDS,
+    REPOS,
+    REPOSET,
+)
+from robottelo.decorators import run_in_one_thread, skip_if_not_set, tier3
+from robottelo.test import UITestCase
+from robottelo.ui.session import Session
+from robottelo.vm import VirtualMachine
+
+
+@run_in_one_thread
+class ContentHostTestCase(UITestCase):
+    """Implements Content Host tests in UI"""
+
+    @skip_if_not_set('clients', 'fake_manifest')
+    def setUp(self):
+        """Create Org, Lifecycle Environment, Content View, Activation key,
+        then create a VM, subscribe it to satellite-tools repo, install
+        katello-ca and katello-agent packages"""
+        super(ContentHostTestCase, self).setUp()
+        self.org_ = entities.Organization().create()
```

Comment:
But, I missed the fact that you maybe don't want the `default organization` so if your intention is to create a random new organization please ignore my comment.